modules/compiler/builtins/cmake/FindBuiltinsTools.cmake

```diff
@@ -29,6 +29,13 @@ include(CAPlatform)  # Bring in CA_HOST_EXECUTABLE_SUFFIX.
 # look for the builtins tools, if it is an empty string, the function will
 # attempt to use eventual in tree targets of the builtins tools.
 function(find_builtins_tools tools_dir)
+  if (CMAKE_CROSSCOMPILING AND OCK_IN_LLVM_TREE)
+    # If we are building OCK in LLVM TREE, ImportLLVM was not included and
+    # so LLVM_PACKAGE_VERSION is not defined.
+    set(LLVM_PACKAGE_VERSION
```

Comment:
Is this just for the check at the bottom? Maybe move it there or add explanation of how the version comes into it.

Reply:
I've moved the check at the bottom